charts/darts-api/values.stg.template.yaml

```diff
@@ -2,3 +2,17 @@
 java:
   image: ${IMAGE_NAME}
   ingressHost: ${SERVICE_FQDN}
+  keyVaults:
+    darts:
+      secrets:
+        - name: GovukNotifyTestApiKey
+          alias: GOVUK_NOTIFY_API_KEY
+  postgresql:
+    enabled: true
```

Comment:
can this be in values.yaml?

Reply:
No as it's creating a postgres pod for the staging pipeline deployment, which we don't want in the main deployment.